src/open_inwoner/userfeed/tests/test_external_tasks.py

```diff
@@ -0,0 +1,115 @@
+from django.contrib.auth import user_logged_in
+from django.test import RequestFactory, TestCase
+
+import requests_mock
+from zgw_consumers.test.factories import ServiceFactory
+
+from open_inwoner.accounts.models import User
+from open_inwoner.accounts.tests.factories import DigidUserFactory
+from open_inwoner.openzaak.models import OpenZaakConfig
+from open_inwoner.openzaak.tests.mocks import ESuiteData
+from open_inwoner.openzaak.tests.shared import FORMS_ROOT
+from open_inwoner.userfeed.choices import FeedItemType
+from open_inwoner.userfeed.models import FeedItemData
+from open_inwoner.userfeed.tests.factories import FeedItemDataFactory
+
+
+class UserFeedExternalTasksTestCase(TestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.user = DigidUserFactory.create(bsn="111111110")
+
+        self.config = OpenZaakConfig.get_solo()
+        self.config.form_service = ServiceFactory(api_root=FORMS_ROOT)
+        self.config.save()
+
+    @requests_mock.Mocker()
+    def test_fetch_tasks_after_login(self, m):
+        request = RequestFactory().get("/dummy")
+        request.user = self.user
+
+        ESuiteData().install_mocks(m)
+
+        user_logged_in.send(User, user=self.user, request=request)
+
+        qs_after_first_login = FeedItemData.objects.all()
```

Comment:
I think you need a list() here to save the actual results because otherwise the queryset is lazily evaluated later and it gets a bit unclear.